docs/meshstack.aks-platform-quickstart-guide.md

```diff
@@ -0,0 +1,223 @@
+
+Make sure you execute the last step of exporting the outputs and save these somewhere. We need them later in this guide.
+
+# Creating the Platform
+
+1. **Open the platform list in the service management area**
+
+   When entering your workspace, click the blue “Go to Services Management” button at the top-right if you haven’t already. Then open the menu on the left-hand side and click on “Platform → Platforms”.
+
+2. **Create a new platform**
+
+   In the list, click on the button “+ Create new platform”. A wizard will open where we will have to enter some information.
+
+3. **Select platform type**
+
+   Under “Platform Type” pick “Azure Kubernetes Service”.
+
+4. **Enter name, identifier & description**
+
+   Enter a fitting name, for example “Azure Kubernetes Service”. You can pick a more specific name if you have multiple clusters for example.
+
+   For the identifier pick something similar to the name, for example `aks`.
+
+   For the description you can write a little bit of additional information about your cluster that people can see in the marketplace. You can also leave this empty for now.
+
+5. **Enter Cloud Platform Web Console URL**
+
+   Enter a URL here that users can use to open the web console of your AKS cluster. You can enter the API Server Address that can be found in Azure Portal when opening up your AKS cluster.
+
+   The support & documentation fields below can be left empty.
+
+6. **Create a new location**
+
+   Locations are a grouping mechanism for multiple platforms in meshStack. For now you can create a new location and give it the name and identifier “AKS”. After that, click “Next”.
+
+7. **Pricing**
+
+   You will be prompted to enter a price for your new platform. You can skip this for now.
+
+
+🎉 The platform is created in meshStack! But it won’t be usable yet. We will have to properly integrate it now
+
+
+# Configuring Integration
+
+1. **Open the Configuration**
+
+   In the last step you should have landed on the configuration page. If not, or if you navigated away again in the meantime, open up the platform and go to “Settings → Configuration”.
+
+2. **Enter base URL**
+
+   Enter the base URL for calling the APIs of your cluster. An example URL is `https://aks-meshcloud-dev-dns-95df19f5.hcp.westeurope.azmk8s.io`.
+
+3. **Add replication config**
+
+   Click the green “+ Add config” button under “Replication Configuration”. This configuration is responsible for managing namespaces & IAM.
+
+4. **Enter access token**
+
+   From your Terraform outputs from before (Preparing Infrastructure step) grab the `replicator_expose_token` value and paste it into the “Access Token” field.
+
+   The next field “Namespace Name Pattern” you can leave as-is for now.
+
+5. **Configure Entra Tenant**
+
+   In the section below under “Additional Access Configuration” go AKS-specific configuration so that meshStack can manage user & groups in your Entra tenant.
```

Comment:
Maybe it improves readability when we provide bullet points when there are more actions required to finish a step